arcgis-ios-sdk-samples/Search/Offline geocode/OfflineGeocodeViewController.swift

```diff
@@ -16,159 +16,144 @@ import UIKit
 import ArcGIS
 
 class GeocodeOfflineViewController: UIViewController, AGSGeoViewTouchDelegate, UISearchBarDelegate, UIAdaptivePresentationControllerDelegate, SanDiegoAddressesViewControllerDelegate {
-    @IBOutlet private var mapView: AGSMapView!
+    @IBOutlet private var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Register self as the touch delgate for the map view.
+            mapView.touchDelegate = self
+            
+            // Add the graphics overlay to the map view.
+            mapView.graphicsOverlays.add(self.graphicsOverlay)
+            // Zoom to San Diego.
+            mapView.setViewpointCenter(AGSPoint(x: -13042254.715252, y: 3857970.236806, spatialReference: .webMercator()), scale: 2e4)
```

Comment:
Not sure if we should use underscores for large numbers here, as these coords don't need to be readable.

Reply:
I don't think there's a need to change this